graphql/schema.graphql

```diff
@@ -797,15 +801,185 @@ type GroupedProjects {
   projects: [Project!]!
 }
 
+type ProjectSettings {
+  githubWebhooksEnabled: Boolean!
+  projectRef: Project
+  vars: ProjectVars
+  aliases: [ProjectAlias]
+  subscriptions: [ProjectSubscription]
+}
+
+type ProjectVars {
+  vars: StringMap
+  privateVars: [String]
+}
+
+type ProjectAlias {
+  id: String!
+  alias: String!
+  gitTag: String!
+  variant: String!
+  task: String!
+  remotePath: String!
+  variantTags: [String!]!
+  taskTags: [String!]!
+}
+
+type ProjectSubscription {
+  id: String!
+  resourceType: String!
+  trigger: String!
+  selectors: [Selector]!
+  regexSelectors: [Selector]!
+  subscriber: SubscriberWrapper
+  ownerType: String!
+  triggerData: StringMap
```

Comment:
Just to double check triggerData is an unpredictable  free form map right? Which is why we have to use to StringMap.

Reply:
triggerData is a map[string]string which I assumed is what StringMap was for. Is this right?

Reply:
https://github.com/evergreen-ci/evergreen/blob/68fd43a436971e7c8df60fdaa322e2f27c8e1c28/model/event/subscriptions.go#L86

Reply:
Yup makes sense 👍